ftw/file/utils.py

```diff
@@ -25,3 +26,70 @@ def redirect_to_download_by_default(context):
             request.other['disable_border'] = border_was_force_disabled
         if border_was_force_enabled:
             request.other['enable_border'] = border_was_force_enabled
+
+
+class FileMetadata(object):
+    """Handles some metadata of an object
+    """
+    def __init__(self, context):
+        self.context = context
+
+    @property
+    def show_author(self):
+        """Checks if the user is anonymous and is not allowAnonymousViewAbout.
```

Comment:
I'd rather tell why I should use the property rather how it does things.
mayb ``True when the file author should be displayed``